bcdaworker/queueing/manager/que.go

```diff
@@ -107,6 +110,24 @@ func (q *queue) processJob(job *que.Job) error {
 		return nil
 	}
 
+	// start a goroutine that will periodically check the status of the parent job
+	go func() {
+		for {
+			select {
+			case <-time.After(15 * time.Second):
+				parentCancelled, _ := q.isParentJobCancelled(jobArgs.ID)
```

Comment:
Can we log this error? Maybe at a warn level to see if this could start causing us grief if we see too many errors.